app/commands/user/reputation_token/award_for_issue.rb

```diff
@@ -26,7 +26,7 @@ def label_change?
       def reputation_level
         # Sort descendingly to award greatest possible reputation
         %i[massive large].find do |type|
-          params[:labels].include?(Github::IssueLabel.for_type(:size, type))
+          params[:labels].include?(Github::IssueLabel.for_type(:rep, type))
```

Comment:
This is the actual change that we needed to make: use `x:rep` for awarding reputation to issue authors.